app/controllers/work_move_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# a controller for changing the collection of work.
+class WorkMoveController < ApplicationController
+  before_action :authenticate_user!
+  verify_authorized except: %i[search]
+
+  def search
+    druid = params[:druid]
+    druid = "druid:#{druid}" unless druid.start_with?('druid:')
```

Comment:
Doesn't the javascript controller always strip off the `druid:` prefix?  Why do we need this?

Reply:
Removed.